crates/python/tests/test_client.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from qcs_sdk import QcsClient
+from qcs_sdk.qpu.client import QcsLoadError
+
+@pytest.fixture
+def default_client_info():
+    return QcsClient().info()
+
+
+@pytest.mark.asyncio
+async def test_client_empty_profile_is_default(default_client_info):
+    """The profile "empty" is configured to be similar to a default client."""
+    client = await QcsClient.load(profile_name="empty")
+
+    assert client.info() == default_client_info
```

Comment:
We could implement `__eq__` for `PyQcsClient` so we don't have to do an equality check on the debug string. Assuming the Rust QcsClient derives eq, we would just return the result of comparing those values.

Reply:
I tried this but ran into https://github.com/PyO3/pyo3/issues/2089 and I don't think it makes sense to implement `PartialOrd` here. Will stick with `__repr__` for now.

Reply:
Ah, good to know! It's not a requirement for me, but curious what you would think about implementing it through `__richcmp__` and using `py.NotImplemented()` for any non equality checking branch, as described [here](https://pyo3.rs/v0.18.0/class/object.html?highlight=__eq__#comparisons)